web/src/pages/Home.tsx

```diff
@@ -1,10 +1,231 @@
+import { NavLink } from "react-router-dom";
+import placeholder from "../assets/placeholder.png";
+import tsujiri from "../assets/tsujiri.jpeg";
+import uadslogo from "../assets/UADSLogo.svg";
+
 export default function Home() {
   return (
-    //Example tailwind
-    <div className="container px-4 pt-24 mx-auto content-center">
-      <h1 className=" text-center text-3xl font-bold text-gray-900 sm:text-4xl dark:text-white">
-        Home Page
+    <>
+      <Landing>
+        <p className="h-20 w-full bg-red-50">Placeholder navbar</p>
+      </Landing>
+
+      <Sponsor />
+
+      <SignupSection />
+
+      <EventSection />
+
+      <ExampleComponent />
+    </>
+  );
+}
+
+function Sponsor() {
+  return (
+    <div className="flex flex-col gap-10 h-screen bg-orange-500">
+      <h1 className=" text-center text-3xl font-bold text-gray-900">
+        Supported by our Sponsors
+      </h1>
+      <SponsorGroups />
+      <PinkButton linkto="/sponsors" buttontext="See our sponsors" />
+    </div>
+  );
+}
+
+function SponsorGroups() {
+  return (
+    <>
+      <div className="flex flex-row flex-nowrap justify-center gap-20">
+        <SponsorCard image={tsujiri} />
+        <SponsorCard2 />
+        <SponsorCard3 />
+      </div>
+    </>
+  );
+}
+
+interface SponsorCardProps {
+  image: string;
+}
+
+function SponsorCard({ image }: SponsorCardProps) {
```

Comment:
I've added the image as an example prop. Try now passing in the shop name text as a parameter. Hint: use the interface SponsorCardProps to determine the type of the parameter